test/e2e/suite-chore/consistency.go

```diff
@@ -180,8 +187,15 @@ var _ = ginkgo.Describe("suite-chore: Consistency between APISIX and the Ingress
 
 		s.NewAPISIXClient().GET("/ip").WithHeader("Host", "httpbin.org").Expect().Status(http.StatusOK)
 	})
-
+	if os.Getenv("K8S_VERSION") == "v1.29.0" {
```

Comment:
We may be able to obtain the k8 version through the API. please ref:
https://pkg.go.dev/k8s.io/client-go@v0.29.0/discovery#DiscoveryClient.ServerVersion


![image](https://github.com/apache/apisix-ingress-controller/assets/79972061/14fc817e-6842-4be9-949a-9b99e2b0d87d)


k8s version >= v1.25